server/controllers/ReportCategoryController.js

```diff
@@ -0,0 +1,200 @@
+import db from '../database/models';
+
+const { ReportCategory } = db;
+
+/**
+ * @class ReportCategoryController
+ *
+ * @export {class}
+ */
+class ReportCategoryController {
+  /**
+   * @description - Create New ReportCategory
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   * @param {object} next call next funtion/handler
+   *
+   * @memberOf ReportCategoryController
+   *
+   * @returns {object} response JSON Object
+   */
+  static createReportCategory(req, res, next) {
+    const { title, description } = req.body;
+
+    return ReportCategory
+      .create({
+        title,
+        description,
+      })
+      .then((newReportCategory) => {
+        res.status(201).send({
+          reportCategory: {
+            message: 'New category created successfully',
+            id: newReportCategory.id,
+            title: newReportCategory.title,
+            description: newReportCategory.description
+          }
+        });
+      })
+      .catch(next);
+  }
+
+  /**
+   * @description - Get a Specific ReportCategory
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   * @param {object} next call next funtion/handler
+   *
+   * @memberOf ReportCategoryController
+   *
+   * @returns {object} response JSON Object
+   */
+  static getReportCategory(req, res, next) {
+    ReportCategory.findOne({
+      where: {
+        id: req.params.id
+      },
+      attributes: ['title', 'description']
+    })
+      .then((reportCategory) => {
+        if (!reportCategory) {
+          return res.status(404).json({
+            errors: {
+              message: 'Report category not found'
+            }
+          });
+        }
+        return res.status(200).json({
+          reportCategory: {
+            title: reportCategory.title,
+            description: reportCategory.description,
+          }
+        });
+      })
+      .catch(next);
+  }
+
+  /**
+   * @description - Get All ReportCategories For a Specific Article
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   * @param {object} next call next funtion/handler
+   *
+   * @memberOf ReportCategoryController
+   *
+   * @returns {object} response JSON Object
+   */
+  static getReportCategories(req, res, next) {
+    const { page, limit, order } = req.query;
+    const offset = parseInt((page - 1), 10) * limit;
+
+    return ReportCategory
+      .findAndCountAll({
+        order: [
+          ['createdAt', order]
+        ],
+        offset,
+        limit,
+      })
+      .then((reportCategory) => {
```

Comment:
I believe pluralizing the "reportCategory" in this case will make more sense as we are likely expecting more than one category.